rsconf/package_data/vm_devbox/main.sh.jinja

```diff
@@ -4,16 +4,12 @@ vm_devbox_main() {
     if vagrant --version > /dev/null 2>&1; then
```

Comment:
Not a sufficient test since vagrant is already installed(?). This can be `&> /dev/null`.
Maybe better to test for the rpm of libvirt?

Reply:
48af956